### DISCUSSION TITLE
MS Teams web works with -T but not otherwise

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#7235] [Q&A]
Since yesterday, if I open MS Teams web app, it freezes after a little bit. More precisely, I think after a few clicks or so. The reaction is that the loading spinner (that was spinning forever while trying to load the selected conversation) freezes, and anything else freeses too; furthermore, if I move to another tab and then back to MS Teams tab, it is completely white.

If I open `qutebrowser -T` then the app seems to behave normally.

I guess I just have to do some cleanup?

---

I've just tried to log out (just takes two clicks), but now when I open it, I don't have a chance to see the login page; all I see, is a white page with the MS Teams logo at the center of it.

**Comment by toofar** [comment]
Hmm, seems to still work for me on Qt6. I don't have an already logged in Qt5 basedir to test with (it works with a new temp one though).

Check in the developer tools and see if you can see any errors in the console or any 403s in the network tab. (I've had issues before with MS when I was signed into an app with one account and then switched accounts in active directory and was getting 403s for some backend services).

FWIW the teams linux app got a new release a few months ago where everything seems to work for once.

**Reply by Aster89** [reply]
There's quite a few errors in the console. Do you know if/how it is possible to dump all that stuff to file? If that's not possible, I'll create upload the stuff on gist.

**Reply by toofar** [reply]
not off hand, no...

**Reply by Aster89** [reply]
[Here](https://gist.github.com/Aster89/a77614a026c5bb4fc951a0a8477ccf2f)'s the gist. Do you see anything that can help?

**Reply by toofar** [reply]
Hmm, I have way more errors than you! (probably due to jmatrix blocking more stuff). The only ones I have in common with you are the CDLWorkerManager AppCache GeoRoutingService ones, so not them.

I would assume that `TypeError: Cannot read property 'put' of null` was relevant. Since it is coming from addOrUpdateUserPreferencesToLocalStore I still assume it is some cookie issues. What is your `content.cookies.accept` set to? If it isn't `all` try setting it to that temporarily and reloading.

Otherwise I guess I would try successively deleting more and more cache to see what worked. All the cookies for teams.microsoft.com (from the developer tools Application tab), then the webengine cache ... then the real heavy option or all browser data, either from the developer tools or the whole webenegine data folder (you'll lose a lot of stuff like preferences for all sites, visited links etc though, I would recommend using a different basedir or the desktop app if losing that stuff doesn't sound good)

**Comment by The-Compiler** [comment]
Could you start with a different `--basedir` and then find out where the culprit is? To do so, run with e.g. `--basedir /tmp/teamstest`, then replace `/tmp/teamstest/data/` with `~/.local/share/qutebrowser/`. If that doesn't trigger it, replace `cache/` by `~/.cache/qutebrowser/`, and finally `config/` by `~/.config/qutebrowser/`. After you found out, if it's `data/`, you might want to start deleting things in `data/webengine/` until you found out what the culprit in there is.

**Reply by Aster89** [reply]
The problem seems to have disappeared. A few hours ago I `pacman -Syu`d, and saw that chromium was being updated. Maybe that's why...
